catalog/urls.py

```diff
@@ -14,4 +14,10 @@
         views.LoanedBooksManageListView.as_view(),
         name="bookinst-manage",
     ),
+    path(
+        "book/<uuid:pk>/renew/", views.renew_book_librarian, name="renew-book-librarian"
+    ),
+    path("author/create/", views.AuthorCreate.as_view(), name="author-create"),
+    path("author/<int:pk>/update/", views.AuthorUpdate.as_view(), name="author-update"),
+    path("author/<int:pk>/delete/", views.AuthorDelete.as_view(), name="author-delete"),
```

Comment:
Gom lại theo từng namespace cùng loại e nhé
```suggestion
    path("books/", views.BookListView.as_view(), name="books"),
    path("book/<int:pk>", views.BookDetailView.as_view(), name="book-detail"),
    path(
        "book/<uuid:pk>/renew/", views.renew_book_librarian, name="renew-book-librarian"
    ),
    path("author/create/", views.AuthorCreate.as_view(), name="author-create"),
    path("author/<int:pk>/update/", views.AuthorUpdate.as_view(), name="author-update"),
    path("author/<int:pk>/delete/", views.AuthorDelete.as_view(), name="author-delete"),
```